fuzz/scripts/run-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+# Number of seconds to run the script. Default to 5 minutes.
+duration="${1:-300}"
+start_time=$SECONDS
+elapsed_time=0
```

Comment:
This should output some kind of message to stderr saying that it's going to run for `n` seconds.